webcompat/static/js/lib/fonts.js

```diff
@@ -0,0 +1,27 @@
+if (/Mobi/.test(navigator.userAgent)) {
+  document.documentElement.className += " font-mobile";
+  var sourceSansPro300 = new window.FontFaceObserver("Source Sans Pro", {
+    weight: 300
+  });
+
+  var sourceSansPro400 = new window.FontFaceObserver("Source Sans Pro", {
+    weight: 400
+  });
+
+  var openSans400 = new window.FontFaceObserver("Open Sans", {
+    weight: 400
+  });
+
+  var openSans600 = new window.FontFaceObserver("Open Sans", {
+    weight: 600
+  });
+
+  Promise.all([
```

Comment:
What about browsers that don't support Promises? This will likely throw in IE, https://caniuse.com/#feat=promises

Reply:
Right I haven't thought about that.
Do you think [jQuery.when()](https://api.jquery.com/jquery.when/) is a good solution?

Reply:
Yeah, that would cover our bases here.